usr/bin/v4l2rtspserver-master.ini

```diff
@@ -0,0 +1,29 @@
+[Configuration]
+#To save some memory you can deactivate the folowing features
+OSD=on
+JPEG=on
+MOTION=on
+
+FontFixedWidth=/usr/fonts/NotoMono-Regular.ttf
```

Comment:
Can we move the fonts into /usr/share/fonts ? that would be one of the standard dirs for fonts on a linux system.

Reply:
OK

Reply:
Perfect. Can the ini file also live in etc? and be a .conf file? for the sake of homegenity...

Reply:
I didn't wanted to do that, because it more an hidden config

Reply:
Renaming *.ini Files (with ini-Format) to .conf seems strange...?

Reply:
okay, then ini it stays. But shouldn't it live in the /etc directory?